src/librustdoc/html/format.rs

```diff
@@ -1280,6 +1272,22 @@ impl PrintWithSpace for hir::Mutability {
     }
 }
 
+crate fn print_constness_with_space(
+    c: &hir::Constness,
+    s: Option<&ConstStability>,
+) -> &'static str {
+    match (c, s) {
+        // const stable or no stability attribute
```

Comment:
Why does this say "or no stability attribute"? Wouldn't that imply it was unstably const?

Reply:
it matches this:
```rust
const fn foo() -> u8 { 42 }
```

`const` functions outside the standard library do not have stability attributes.

Reply:
Ah ok, you mean if `feature(staged_api)` isn't set. Can you say that here? I was confused at first because I think you can have functions without a stability attribute if it's inherited from the parent module.